### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/intro/intro.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#782] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/intro/intro.html

主题初衷 VuePress 在一定程度上扩展了 Markdown 语法，但仍然缺少一些常用的功能，例如文本对齐、标记、流程图、公式、演示等，同时默认主题提供的一些功能较弱或缺失，如图片预览、代码块复制、目录页等。同时 VuePress 默认主题太简陋，功能不够强大。在这种情况下，vuepress-theme-hope 和一些系列插件就应运诞生。 与默认...

https://theme-hope.vuejs.press/zh/guide/intro/intro.html

**Comment by zhh2001** [comment]
**你好，这个主题以后会移植VitePress吗**

**Reply by Mister-Hope** [reply]
暂无计划，VitePress 和 VuePress 存在巨大差异，大量功能无法在 VitePress 下运行。

**Comment by BraveJJJ** [comment]
544

**Comment by DUDU-dudu-DUDU** [comment]
怎么更换主页背景图呀

**Reply by Mister-Hope** [reply]
- https://theme-hope.vuejs.press/zh/guide/layout/home.html#%E7%AB%99%E7%82%B9%E4%BF%A1%E6%81%AF
- https://theme-hope.vuejs.press/zh/guide/blog/home.html#%E5%8D%9A%E5%AE%A2%E7%B1%BB%E5%9E%8B%E4%B8%BB%E9%A1%B5

如果你的下一个评论仍然可以很轻松通过搜索框或浏览侧边栏轻松定位的话，我会忽略它。不要**试图通过浪费他人时间**来节省你的。

**Reply by DUDU-dudu-DUDU** [reply]
十分抱歉，刚才真的找了好久，快哭了。万分感谢

**Reply by Mister-Hope** [reply]
善用搜索...... 明明你搜背景图就搜的出来

**Comment by kittors** [comment]
默认是英文，怎么改成默认是中文显示？
